src/main/java/org/moddingx/modlistcreator/modlist/README.md

```diff
@@ -0,0 +1,73 @@
+# Modlist
```

Comment:
Please move to a `docs` directory in repository root, just as [ModGradle](https://github.com/ModdingX/ModGradle/tree/master/docs) does.